src/js/utils/StringUtil.js

```diff
@@ -215,6 +215,16 @@ const StringUtil = {
     }, []);
 
     return this.idToTitle(splitID, splitBy, replace, removeConsecutive);
+  },
+
+  lrtrim(str, trimStr = " ") {
```

Comment:
Native JS `.trim()` will trim all whitespace characters (space, tab, no-break space, etc.) and all the line terminator characters (LF, CR, etc.). We could simulate this with `\s\uFEFF\xA0` - what do you think about changing to `trimStr = "\s\uFEFF\xA0"`?